src/structures.jl

```diff
@@ -91,12 +94,13 @@ mutable struct GraphAssetData
         max_energy_timeframe_partition,
         min_energy_timeframe_partition,
     )
-        timeframe_profiles = Dict{String,Vector{Float64}}()
-        rep_periods_profiles = Dict{Tuple{String,Int},Vector{Float64}}()
-        timeframe_partitions = PeriodsBlock[]
-        rep_periods_partitions = Dict{Int,Vector{TimestepsBlock}}()
+        timeframe_profiles = Dict{Int,Dict{String,Vector{Float64}}}()
+        rep_periods_profiles = Dict{Int,Dict{Tuple{String,Int},Vector{Float64}}}()
+        timeframe_partitions = Dict{Int,PeriodsBlock}()
```

Comment:
I haven't looked at the whole code, but I think this should be the following:
```suggestion
        timeframe_partitions = Dict{Int,PeriodsBlock[]}()
```